src/tests/Resolution.test.ts

```diff
@@ -1142,6 +1490,58 @@ describe('Resolution', () => {
       expectSpyToBeCalled(spies);
       expect(isRegistered).toBe(false);
     });
+    skipItInLive(
```

Comment:
@sudoryan It's better no skip test in live if possible. I believe it's possible to create live version of all tests that were market as `skipItInLive` in this pull requests. Please remove all `skipItInLive` usages and let me know if you have any issues with minting test domains for live tests.